docs/components/modeler/desktop-modeler/plugins/plugins.md

```diff
@@ -193,7 +193,7 @@ npm run build
 
 When creating a plugin, you can place the directory containing your plugin in the aforementioned `resources/plugins` directory.
 
-Plugins will be loaded on application startup (menu plugins) or reload (style and modeling tool plugins). To reload the application, open the developer tools F12 and press `CtrlOrCmd + R`. This will clear all unsaved diagrams.
+Plugins will be loaded on application startup (menu plugins) or reload (style and modeling tool plugins). To reload the application, open the developer tools F12 and press `CtrlOrCmd+R`. This will clear all unsaved diagrams.
```

Comment:
This one's a little odd to me - I know it's what was there already, but do we want it to read as "**`Ctrl+R` or `Cmd+R`**" ?

Reply:
As @conceptualshark mentions, I think this was originally meant to read as `Ctrl+R` or `Cmd+R`
```suggestion
Plugins will be loaded on application startup (menu plugins) or reload (style and modeling tool plugins). To reload the application, press `F12` to open the developer tools, and press `Ctrl+R` or `Cmd+R`. This will clear all unsaved diagrams.
```